src/script/printer/tir/var.cc

```diff
@@ -16,3 +16,47 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+#include <tvm/node/functor.h>
+#include <tvm/runtime/data_type.h>
+#include <tvm/tir/expr.h>
+#include <tvm/tir/op.h>
+#include <tvm/tir/var.h>
+
+#include "../util.h"
+#include "./utils.h"
+
+namespace tvm {
+namespace script {
+namespace printer {
+
+ExprDoc PrintVar(tir::Var v, IRDocsifier p) {
+  Optional<ExprDoc> var_doc = p->sym->GetObjectDoc(v);
+  if (var_doc == nullptr) {
+    auto frame = p->GetFrame<VarDefFrame>().value();
+    IdDoc free_var_doc = frame->DefByName(v, p->sym->GetUniqueName(v->name_hint));
+    return free_var_doc;
+  } else {
+    return var_doc.value();
+  }
+}
+
+TVM_STATIC_IR_FUNCTOR(IRDocsifier, vtable).set_dispatch<tir::Var>(PrintVar);
+TVM_STATIC_IR_FUNCTOR(IRDocsifier, vtable).set_dispatch<tir::SizeVar>(PrintVar);
+
+ExprDoc PrintIterVar(tir::IterVar v, IRDocsifier p) {
+  ExprDoc dom_arg = LiteralDoc::None();
```

Comment:
Make this unsupported.

Add helper function in tir.h to translate for specific cases

```
Doc VarDef(tir::Var); // a = T.var("int32")
Doc VarDecl(tir::Var); // a: T.int32
Doc IterVarBlockVarDef(tir::IterVar); // a = T.axis.S/R(...)
Doc IterVarLaunchThreadDef(tir::IterVar); // a = T.launch_thread(...)
Doc BufferMatchBuffer(tir::Buffer, BufferPrintInfo); // a = T.match_buffer(...)
Doc BufferDef(tir::Buffer, BufferPrintInfo); // a = T.buffer_decl(...)
Doc BufferDecl(tir::Buffer, BufferPrintInfo); // a: T.Buffer()
```